common/pdf-jpg-export-js/PdfExportJS/Services/Drawing.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PdfExportJS.Services
+{
+    public class Drawing
```

Comment:
What about naming this `DrawingService` instead as the `Service` suffix is commonly used in such cases?